### DISCUSSION TITLE
Default display text format problem

### DISCUSSION THREAD

**Discussion by TrungLeMD** [RyotaUshio/obsidian-pdf-plus#256] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/df8acd07-bc0b-4557-b90d-19f426d05f16)
I noticed that a user on Windows can use the "Create new note" feature of PDF++ in Obsidian, but it is currently unavailable on MacOS.
![CleanShot 2024-08-21 at 01 02 47](https://github.com/user-attachments/assets/97d7b0a8-d678-4701-83b0-74bcab0ee9c2)

**Comment by TrungLeMD** [comment]
![image](https://github.com/user-attachments/assets/487bccbe-a539-4306-b673-0321525e8d24)

**Comment by RyotaUshio** [comment, marked as answer]
The copy format called "Create new note" was one of the default formats back when this plugin was in its very early phase of development. Then, it was removed because it's a bit too complex and might be overwhelming for some users.
That's why this format is not set by default if you install PDF++ in your vault recently. (in other words, OS doesn't matter)

Probably you installed PDF++ on Windows first (like half a year ago?) and then recently installed it on Mac too, didn't you?

You can still use the format by manually adding it.
